src/index.hot.js

```diff
@@ -1,19 +1,20 @@
+/* eslint global-require: 0 */
+import 'babel-polyfill';
```

Comment:
> For develop mode, I think I should keep which are consistent?

Indeed, this is a tough question. Consistency is important, but if consistency doesn't make sense, should we still follow consistency?